seqr/views/apis/users_api.py

```diff
@@ -24,7 +24,7 @@
 @login_required(login_url=API_LOGIN_REQUIRED_URL)
 def get_all_collaborator_options(request):
     collaborators = set()
-    for project in get_projects_user_can_view(request.user):
+    for project in get_local_access_projects(request.user):
```

Comment:
this endpoint is only ever used for giving a list of users who can be added to a project as local collaborators. Since this functionality is completely disabled for projects that have workspaces associated with them, there is no harm in restricting the user list here to only local projects.